tests/components/grid-jsx/Grid.vue

```diff
@@ -95,6 +95,8 @@ export default {
     const { sortKey, capitalize } = this
     return (
       <table class="grid">
+        {/** @slot Use this slot header */}
+        <slot name="header" />
```

Comment:
Perhaps also add a method/computed property with the footer slot, and then include it below. See my comment above